torch/csrc/jit/codegen/cuda/fusion.cpp

```diff
@@ -376,7 +376,8 @@ void Fusion::printMath(bool from_outputs_only) {
 std::vector<Val*> Fusion::inputsAndCreated() {
   auto result = inputs_;
   for (auto expr : exprs()) {
-    if (expr->inputs().empty()) {
+    auto tv_inputs = ir_utils::filterByType<TensorView>(expr->inputs());
+    if (tv_inputs.empty()) {
```

Comment:
Why are you filtering out scalar expressions?

Reply:
For example
```
T0 = arange(start, stop, step)
```
Then T0 is a tensor created inside the fusion. But it has three inputs: `start`, `stop`, and `step`.